dev-tools/omdb/tests/test_all_output.rs

```diff
@@ -274,27 +357,54 @@ async fn do_run<F>(
 ) where
     F: FnOnce(Exec) -> Exec + Send + 'static,
 {
-    do_run_extra(output, modexec, cmd_path, args, &ExtraRedactions::new())
-        .await;
+    do_run_extra(
+        output,
+        modexec,
+        cmd_path,
+        args,
+        true,
+        Some(&ExtraRedactions::new()),
+    )
+    .await;
+}
+
+async fn do_run_no_redactions<F>(
+    output: &mut String,
+    modexec: F,
+    cmd_path: &Path,
+    args: &[&str],
+) where
+    F: FnOnce(Exec) -> Exec + Send + 'static,
+{
+    do_run_extra(output, modexec, cmd_path, args, false, None).await;
 }
 
 async fn do_run_extra<F>(
     output: &mut String,
     modexec: F,
     cmd_path: &Path,
     args: &[&str],
-    extra_redactions: &ExtraRedactions<'_>,
+    redactions: bool,
+    extra_redactions: Option<&ExtraRedactions<'_>>,
```

Comment:
It seems like `redactions == extra_redactions.is_some()`, is that right? If that's the case, we should probably pass only the `extra_redactions` argument, and use its value internally to decide what to do.

Reply:
@bnaecker these are different from what I can tell. Extra redactions are ones beyond the basic ones.

Reply:
yep. derp.